tawk_to.module

```diff
@@ -60,7 +43,7 @@ function tawk_to_page_build(&$page)
     $widget_id = $widget['widget_id'];
 
     // get visibility options
-    $options = variable_get(TAWKTO_WIDGET_OPTIONS, '');
+    $options = config_get('tawk_to.settings','widget_options');
```

Comment:
We may also want to add the config name to the constants declaration. Something like `define('TAWKTO_CONFIG_NAME', 'tawk_to.settings');` so we can just define it once.

Reply:
@alvinjohnsonso I'm not quite sure of the benefits of doing this.  The constants will be longer than the strings they're replacing.  Please can you explain why this is better practice?

Reply:
Thanks for the explanation. Done